README.md

```diff
@@ -147,26 +147,28 @@ Once you've done that, restart the IDE.
 
 ## Run Locally
 
-* Build and run tests
+* Build and create jar:
 
 ```shell
-./gradlew clean build
+./gradlew clean bootJar
 ```
 
-**Note**: Make sure you have docker daemon running locally to be able to run integration test, by
-execute ``docker ps``
-
-* Create Jar
+* Start Docker Compose
 
 ```shell
-./gradlew clean bootJar
+  docker compose -f docker/docker-compose.yml up --build
 ```
 
-* Start database
+This will create two Docker instances in your Docker desktop:
 
-```shell
-docker run --pull always --name surrealdb -p 8000:8000 -d surrealdb/surrealdb:latest start
-```
+1. the SurrealDB database
+2. the WCC backend app
```

Comment:
You can add another step to stop the container of the application in case you want to run via IntelliJ and debug and etc.